inc/command/class-command.php

```diff
@@ -485,7 +487,12 @@ protected function codecept( InputInterface $input, OutputInterface $output ) {
 			}
 
 			$output->write( '<info>Running CodeCeption..</info>', true, $output::VERBOSITY_NORMAL );
-			$return = $this->run_command( $input, $output, 'vendor/bin/codecept', $options );
+			$return = $return ?: $this->run_command( $input, $output, 'vendor/bin/codecept', $options );
+
+			// Stop executing rest of the suites if one fails and `continue` argument is absent.
+			if ( $return && ! $input->getOption( 'continue' ) ) {
```

Comment:
Missed this before but codeception accepts a comma separated list of suites, probably don't need the loop or this option, e.g. `composer dev-tools codecept run acceptance,functional,unit`.

Reply:
Codeception recommends _against_ running multiple suites in the same thread, hence most of the custom handling here, otherwise we would proxy the `run` command as is without any special handling. The way we did it here means end users can still use comma-separated suite names, but they'll work one by one, with each one checking for dependencies ( database import, browser container ) to minimize redundant actions.

Reply:
Sorry you did say that. As long as we only hit this loop with the `run` command we should be good